src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -582,6 +582,71 @@ void cannotCreateTlrRequestWhenUserHasLoanForSomeItemsOfInstance() {
       hasParameter("userId", usersFixture.jessica().getId().toString()))));
   }
 
+  @Test
+  void cannotCreateTlrRequestWhenUserHasAlreadyRequestedInstanceOrItemTlrOnRequestLevelTitle() {
+    reconfigureTlrFeature(TlrFeatureStatus.ENABLED);
+
+    IndividualResource instanceMultipleCopies = instancesFixture.basedUponDunkirk();
+    final UUID pickupServicePointId = servicePointsFixture.cd1().getId();
+    UUID instanceId = instanceMultipleCopies.getId();
+
+    buildItem(instanceId, UUID.randomUUID().toString());
+    requestsClient.attemptCreate(makeRepresentation(pickupServicePointId, instanceId));
+
+    Response postResponse = requestsClient.attemptCreate(makeRepresentation(pickupServicePointId, instanceId));
+
+    assertThat(postResponse, hasStatus(HTTP_UNPROCESSABLE_ENTITY));
+    assertThat(postResponse.getJson(), hasErrorWith(allOf(
+      hasMessage("This requester already has an open request for this instance"),
+      hasParameter("requesterId", usersFixture.james().getId().toString()),
+      hasParameter("instanceId", instanceId.toString()))));
+  }
+
+  private JsonObject makeRepresentation(UUID pickupServicePointId, UUID instanceId) {
+    return new RequestBuilder()
+      .hold()
+      .withPickupServicePointId(pickupServicePointId)
+      .titleRequestLevel()
+      .withInstanceId(instanceId)
+      .withNoItemId()
+      .withNoHoldingsRecordId()
+      .by(usersFixture.james())
+      .create();
+  }
+
+  @Test
+  void cannotCreateTlrRequestWhenUserHasAlreadyRequestedInstanceOrItemTlrOnRequestLevelItem() {
+    reconfigureTlrFeature(TlrFeatureStatus.ENABLED);
+
+    final ItemResource item = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource requestPickupServicePoint = servicePointsFixture.cd1();
+    final IndividualResource charlotte = usersFixture.charlotte();
+    final IndividualResource jessica = usersFixture.jessica();
+
+    checkOutFixture.checkOutByBarcode(item, charlotte);
+    requestsClient.create(makeRequestBuilder(item, requestPickupServicePoint, jessica));
+    final Response response = requestsClient.attemptCreate(
+      makeRequestBuilder(item, requestPickupServicePoint, jessica));
+
+    assertThat(response, hasStatus(HTTP_UNPROCESSABLE_ENTITY));
+    assertThat(response.getJson(), hasErrors(1));
+    assertThat(response.getJson(), hasErrorWith(allOf(
+      hasMessage("This requester already has an open request for one of the instance's items"),
+      hasParameter("itemId", item.getId().toString()),
+      hasParameter("requesterId", jessica.getId().toString()),
+      hasParameter("instanceId", item.getInstanceId().toString()))));
+  }
+
+  private RequestBuilder makeRequestBuilder(ItemResource item, IndividualResource requestPickupServicePoint,
```

Comment:
I think you don't have to pass requestPickupServicePoint from the outside

Reply:
done